pandas/io/formats/format.py

```diff
@@ -108,7 +108,7 @@
 )
 from pandas.io.formats.printing import (
     adjoin,
-    justify,
+    justify as jst,
     pprint_thing,
 )
```

Comment:
`jst` isn't so obvious...how about
```
from pandas.io.formats import printing
```
and then in the code, use `printing.adjoin`, `printing.justify`, and `printing.pprint_thing`?